python/ml4ir/applications/ranking/model/metrics/helpers/metrics_helper.py

```diff
@@ -16,9 +17,83 @@ class RankingConstants:
     DIFF_MRR = "diff_MRR"
     OLD_ACR = "old_ACR"
     NEW_ACR = "new_ACR"
+    OLD_NDCG = "old_NDCG"
+    NEW_NDCG = "new_NDCG"
     TTEST_PVALUE_THRESHOLD = 0.1
 
 
+def add_top_graded_relevance_record_column(df, label_col, new_col_name):
+    """
+    Adds a new column indicating the top graded relevance record per query in the DataFrame.
+
+    Args:
+        df (pandas.DataFrame): The input DataFrame.
+        label_col (str): The column name containing the relevance scores.
+        new_col_name (str): The name for the new column indicating the top graded relevance.
+
+    Returns:
+        pandas.DataFrame: The input DataFrame with the new column added.
+        """
+    # remove queries with no relevance scores
+    grouped_sum = df.groupby('query_id')[label_col].sum()
```

Comment:
@mohazahran Do not hardcode the feature names

Reply:
done